cmd_push.go

```diff
@@ -205,13 +206,34 @@ func pushWorker(sps *sharedPushState) {
 				sps.setFatalError(err)
 				return
 			}
-
+			ignoreTableRegex := t.Dir.Config.Get("ignore-table-regex")
+			re, err := regexp.Compile(ignoreTableRegex)
+			if err != nil {
+				sps.setFatalError(fmt.Errorf("Invalid regular expression on ignore-table-regex: %s; %s", ignoreTableRegex, err))
+				return
+			}
```

Comment:
This regex creation runs within a loop, and always performs the same operation. It can be extracted outside the loop.
Also, it's the same creation as in https://github.com/skeema/skeema/pull/15/files#diff-1b3966e75764b7531e27b460e0f21b06R181, so you may as well put the regex in a globally accessible place, or otherwise create a single function that creates & returns it.

Reply:
The function, assuming one is written, would return `nil` for the regex.

Reply:
The loop it is in assign `t` each time and ignoreTableRegex uses `t`, so I don't think I can move it out.

Reply:
I stand corrected.